frontend/server/src/DAO/CourseIdentityRequest.php

```diff
@@ -12,4 +12,72 @@
  * @access public
  */
 class CourseIdentityRequest extends \OmegaUp\DAO\Base\CourseIdentityRequest {
+    /**
+     * @return list<array{accepted: bool|null, admin_id: int, identity_id: int, last_update: null|string, course_id: int, request_time: string}>|null
+     */
+    public static function getFirstAdminForCourseRequest(
```

Comment:
```suggestion
    public static function getRequestsForCourseWithFirstAdmin(
```

ay madre, me tomó mucho tiempo entender qué era lo que estaba intentando hacer esta función. primero pensé que sólo se quería obtener el primer administrador para el curso, pero en realidad lo que quiere hacer es obtener el primer administrador _para cada petición_.

Reply:
Listo